stream_alert_cli/config.py

```diff
@@ -17,24 +17,26 @@
 import json
 import os
 import re
-import sys
 
 from stream_alert.shared import metrics
 from stream_alert_cli.helpers import continue_prompt
 from stream_alert_cli.logger import LOGGER_CLI
 
 
+class CLIConfigError(Exception):
+    pass
+
 class CLIConfig(object):
-    """Provide an object to load, modify, and display the StreamAlertCLI Config"""
-    def __init__(self):
-        self.config_files = OrderedDict([
-            ('global', 'conf/global.json'),
-            ('lambda', 'conf/lambda.json')
-        ])
-        self.config = self.load()
+    """A class to load, modify, and display the StreamAlertCLI Config"""
```

Comment:
*For future consideration*: we should think about converting this class to subclass `dict` directly and we wouldn't have to do a lot of the extra magic like `__getitem__` overrides, etc. This might also help with writing the config to disk every time `__setitem__` is called.

Reply:
I'm fine with this